scripts/utils.sh

```diff
@@ -147,3 +147,22 @@ function get_ipv4_address {
     echo "${hipv4}"
     return 0
 }
+
+function verify_nvidia-smi {
+    if command_exists "nvidia-smi"; then
+        nvidia-smi --version
```

Comment:
```suggestion
        # We are careful here in case we are running in a container and LD_LIBARY_PATH has been wiped
        LD_LIBRARY_PATH="/.singularity.d/libs:${LD_LIBRARY_PATH}" nvidia-smi --version
```